profiler/test/Datadog.Profiler.IntegrationTests/Bugs/UnsafeToUnwindTest.cs

```diff
@@ -0,0 +1,103 @@
+// <copyright file="UnsafeToUnwindTest.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2022 Datadog, Inc.
+// </copyright>
+
+using System.Linq;
+using Datadog.Profiler.IntegrationTests.Helpers;
+using FluentAssertions;
+using Xunit.Abstractions;
+
+namespace Datadog.Profiler.IntegrationTests.Bugs
+{
+    public class UnsafeToUnwindTest
+    {
+        private readonly ITestOutputHelper _output;
+
+        public UnsafeToUnwindTest(ITestOutputHelper output)
+        {
+            _output = output;
+        }
+
+        [TestAppFact("Samples.Computer01")]
+        public void AvoidUnwindingUnsafeExecution(string appName, string framework, string appAssembly)
+        {
+            var runner = new TestApplicationRunner(appName, framework, appAssembly, _output, commandLine: "--scenario 28", enableTracer: true);
+            runner.Environment.SetVariable(EnvironmentVariables.SkippedMethods, "Samples.Computer01.UnsafeToUnwind[Wrap_UnSafeToUnwind];Samples.Computer01.UnsafeToUnwind[Wrap_RaiseExceptionUnsafeUnwind];Samples.Computer01.UnsafeToUnwind[Wrap_ContentionUnsafeToUnwind]");
```

Comment:
Maybe add another test to validate the different possibilities in term of syntax like type1[foo, bar] in addition to type1[foo];type1[bar]. Is there any log when a given method does not exist (i.e. detect typo)?

Reply:
/cc @gleocadie